### ISSUE TITLE
I can't access my Android phone's internal storage in Nitrux 2.7.0

### ISSUE THREAD

**Issue by sprc9034** [Nitrux/nitrux-bug-tracker#97]
When I connect my Android phone to my PC via type-c cable and turn on the file transfer mode, my phone shows up in index but, when I click it it shows location not found.

**Comment by UriHerrera** [comment]
What phone is it?

**Comment by sprc9034** [comment]
> What phone is it?

Oppo find x5 lite ( works fine with other distros)

**Comment by UriHerrera** [comment]
> > What phone is it?
> 
> Oppo find x5 lite ( works fine with other distros)

I'm not sure what other distributions do, but this bug with KIO seems to be the problem: https://bugs.kde.org/show_bug.cgi?id=412257.

One of the comments mentions that killing the process `kiod5` allows the storage to be accessed.

So before anything, I tried with a Motorola G52, plugged it in, and selected File Transfer Mode in the Android popup.

Then, I opened Station and, using a container, installed the package providing `mtp-detect` (I have a Debian container, so it's `mtp-tools`), and indeed `mtp-detect` says the device is busy.

![image](https://user-images.githubusercontent.com/3053525/227176084-0f8916b8-dd19-4551-a3d3-3770624e9844.png)

Then, I killed the KIO process on the host, so I did `killall kiod5` and ran `mtp-detect` on the container once again. The device is now detected correctly.

![image](https://user-images.githubusercontent.com/3053525/227176436-9dbc76ac-ea2e-4446-9c92-a11ee54415fe.png)

I also used an Android File Transfer application, as with `mtp-detect`; plugging the phone and opening the app only displayed a window with an error message _No MTP Device found_. Once I have done the above, I can access the phone's storage using the app.

![image](https://user-images.githubusercontent.com/3053525/227177056-c16539fe-a30f-4ed6-a5fd-e8b4d51fffdc.png)

It's worth mentioning that I didn't have to add any rule for udev; I only did the above steps.
